package Methods;

public class Exception1 {
	public static void main(String[] args)
	{
		try(int a=8, int b=0);
	         BufferedReader br = new BufferedReader(fr)) {
		{
	
			int num = 100/0;
			System.out.println("Divided by zero"+num);
		}
		catch(ArithmeticException e)
		{
			e.printStackTrace();
		}
		finally
		{
			System.out.println("Continue Rest code");
		}
		
	}

}

